bom/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+plugins {
+    `java-platform`
+    `kord-publishing`
+}
+
+javaPlatform {
+    allowDependencies()
+}
+
+val me = project
+rootProject.subprojects {
+    if (name != me.name) {
+        me.evaluationDependsOn(path)
+    }
+}
+
+dependencies {
+    constraints {
+        rootProject.subprojects.forEach { subproject ->
+            if (subproject.plugins.hasPlugin("maven-publish") && subproject.name != name) {
```

Comment:
What is this block of code for

Reply:
It adds every kord module which applies the maven-publish plugin and is not the bom module as a contraint dependency.
it's also possible to add every module by hand